src/ralph/admin/autocomplete.py

```diff
@@ -35,7 +36,9 @@ def get(self, request, *args, **kwargs):
         """
         Returns serialized dict as JSON object.
         """
-        can_edit = ralph_site._registry[self.model].has_change_permission(
+        can_edit = not issubclass(
```

Comment:
it's not good condition IMO - sometimes we'll point in autocomplete to concrete subclass of Polimorphic (BaseObject), ex. Database will have reference to physical server (DatacenterAsset) on which it's stored (it's example, of course), and with your condition we won't be able to edit DatacenterAsset from Database view.